quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java

```diff
@@ -76,11 +76,18 @@ public static void reset() {
     }
 
     public static ConfigValue getValue(ConfigSourceInterceptorContext context, String name) {
+        name = removeProfilePrefixIfNeeded(name);
         PropertyMapper<?> mapper = getMapper(name);
-        // during re-aug do not resolve the server runtime properties and avoid they included by quarkus in the default value config source
-        if ((isRebuild() || Environment.isRebuildCheck()) && isKeycloakRuntime(name, mapper)) {
+
+        // During re-aug do not resolve the server runtime properties and avoid they included by quarkus in the default value config source.
+        //
+        // The special handling of log properties is because some logging runtime properties are requested during build time
+        // and we need to resolve them. That should be fine as they are generally not considered security sensitive.
+        // See https://github.com/quarkusio/quarkus/pull/42157
+        if ((isRebuild() || Environment.isRebuildCheck()) && isKeycloakRuntime(name, mapper) && !name.startsWith("quarkus.log.")) {
```

Comment:
Alternative to this would be introducing a third option kind – to runtime and buildtime, we'd need to add something like runtimeAvailableAtBuildtime. Given this currently affects logging option and the alternative would be lot of boilerplate code, this seemed like a better option.

Reply:
Does this mean that the logging properties need to be captured / compared in the persisted config?